src/lem.lisp

```diff
@@ -122,6 +124,12 @@ Options:
                                      (error "Please, specify a filename to log to."))
                                    (setf (command-line-arguments-log-filename parsed-args)
                                          filename)))
+                                ((member arg '("-i" "--interface") :test #'equal)
+                                 (let ((interface (pop args)))
+                                   (unless (and interface (/=  0 (array-total-size interface)))
+                                     (error "Please, specify an interface to use."))
+                                   (setf (command-line-arguments-interface parsed-args)
+                                         (intern (string-upcase interface) "KEYWORD"))))
```

Comment:
Also `alexandria:make-keyword` (with the string upcase)

> Interns the string designated by NAME in the KEYWORD package.

but if this works it works.

Reply:
I switched it to `alexandria:make-keyword` in a commit I just pushed to make it more clear what is happening to someone skimming the code fast.